CMakeLists.txt

```diff
@@ -31,7 +31,7 @@ add_definitions(${LIBXML2_DEFINITIONS})
 # cURL
 find_package(CURL REQUIRED)
 include_directories(${CURL_INCLUDE_DIRS})
-add_definitions(${LIBXML2_DEFINITIONS})
+add_definitions(${CURL_DEFINITIONS})
```

Comment:
This doesn't look relevant. If it was necessary, we should break it into a separate commit with an explanation why

Reply:
Sure thing, agreed it should be split out-- I was touching these files while working on this, actually :innocent:.  First pass used glib, so was detecting that and plumbing flags accordingly.

Anyway this is a minor fixup, split off into #80 .